site/src/components/table.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Table
+description: Table element and it's modifiers.
+template: standard
+---
+
+# Table
+
+Table element helps organize content into rows and columns. `table` class adds basic styles to the tables fast and easily. Adding supported modifiers will get a custom and responsive look to the table.
+
+## Usage
+
+Here is an example of applying `table` class to the table element:
+
+```html
+<table class="table">
+	<caption>
+		Table caption
+	</caption>
+	<thead>
+		<tr>
+			<th>Heading row</th>
+			<th>Heading cell</th>
+			<th>Heading cell</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>Row one</td>
+			<td>Cell</td>
+			<td>Cell</td>
+		</tr>
+		<tr>
+			<td>Row two</td>
+			<td>Cell</td>
+			<td>Cell</td>
+		</tr>
+		<tr>
+			<td>Row three</td>
+			<td>Cell</td>
+			<td>Cell</td>
+		</tr>
+	</tbody>
+	<tfoot>
+		<tr>
+			<th>Footer row</th>
+			<th>Cell</th>
+			<th>Cell</th>
+		</tr>
+	</tfoot>
+</table>
+```
+
+## Modifiers
+
+These classes are available to style tables in different ways, feel free to mix and match them!
+
+| Class                      | Usage                                               |
+| -------------------------- | --------------------------------------------------- |
+| `is-striped`               | Adds background color to every odd row              |
+| `is-inner-borders`         | Adds inner borders to the table                     |
+| `is-header-background`     | Adds background color to the header row             |
+| `is-row-header-background` | Adds background color to the first cell of each row |
+| `is-stacked-mobile`        | Stacks table cells on mobile screen sizes.          |
```

Comment:
Let's reduce this list to only `is-stacked-mobile`.

Reply:
table-stacked-mobile

Reply:
Removed that table completely, since only one modifier left. Will add it back when we'll add more modifiers